src/view/surface.cpp

```diff
@@ -438,6 +438,10 @@ void wf::wlr_surface_base_t::_simple_render(const wf::framebuffer_t& fb,
     OpenGL::render_begin(fb);
     OpenGL::render_texture(texture, fb, geometry, glm::vec4(1.f),
         OpenGL::RENDER_FLAG_CACHED);
+    if (fb.scale - floor(fb.scale) < 0.001)
```

Comment:
Can you please add a comment in the source code explaining why we do this?

Reply:
he said it's because it only does nearest when it's an integer
which makes sense to me

Reply:
It may make sense now because we know what problem this PR attempts to solve. It may be much less clear if someone looks at this code in 2 years or so.

Reply:
Updated.

@CrezyDud s/he/she/